monkey/infection_monkey/exploit/smbexec.py

```diff
@@ -129,7 +131,7 @@ def _exploit_host(self):
         resp = scmr.hRCreateServiceW(scmr_rpc, sc_handle, self._config.smb_service_name, self._config.smb_service_name,
                                      lpBinaryPathName=cmdline)
         service = resp['lpServiceHandle']
-
+        T1035Telem(ScanStatus.USED, "SMB exploiter ran the monkey by creating a service via MS-SCMR.").send()
```

Comment:
@danielguardicore Is it possible that monkey won't be able to create services?

Reply:
Technically (Almost) all windows APIs can fail, so yes. There are several things that can go wrong like network issues.
Any way, let's determine the service was executed successfully iff `scmr.hRStartServiceW(scmr_rpc, service)` didn't raise an exception

Reply:
`scmr.hRStartServiceW(scmr_rpc, service)` raised an exception, but the machine was still exploited successfully. I think we should send `UNSCANNED` if this exploiter returns false, else send `USED`

Reply:
Which exception was raised? I'd rather miss some successful reports than reporting something wrong

Reply:
I don't remember at this point, but clearly exceptions here are expected, thus `pass` is used, not `return False`. So either **a**. SMB exploiter is bugged: we show exploited when it's not or **b**. we should send telems according to exploiter: used if it returns True, scanned if it returns False. I think b is more likely.

Reply:
I replied on a DM. Keep it as is right now. Open an issue about it